content/blog/2016/2016-07-26-blueocean-alpha-4.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Jenkins Blue Ocean 1.0-alpha-4 released"
+tags:
+- blueocean
+- ux
+- pipeline
+author: i386
+---
+
+We've released Blue Ocean 1.0-alpha-4 to the experimental update center.
```

Comment:
We have not typically received blog posts with plugin changelogs/updates, outside of one or two from @jglick on major/notable Pipeline changes.  It's not clear from this blog post what's notable to an end-user or somebody following the Blue Ocean project.


Perhaps a screenshot or more description of the release from @michaelneale would help fill this out?